pkg/ingester/circuitbreaker_test.go

```diff
@@ -455,7 +455,7 @@ func TestIngester_PushToStorage_CircuitBreaker(t *testing.T) {
 				failureThreshold := 2
 				var initialDelay time.Duration
 				if initialDelayEnabled {
-					initialDelay = 200 * time.Millisecond
+					initialDelay = time.Hour
 				}
```

Comment:
There is also `TestIngester_Push_CircuitBreaker_DeadlineExceeded` down there, which uses the same test pattern, where `initialDelay = 200 * time.Millisecond`. Should we, maybe, update it also, to keep the tests consistent. Otherwise, it makes it even more complicated to follow what those tests are actually testing.

Reply:
yeah probably. I think one uses ingest storage, the other one doesn't

Reply:
pushed this https://github.com/grafana/mimir/pull/9351/commits/a299bcd6857573a73580285d071ba72a4b42604c